tests/execution.rs

```diff
@@ -3435,6 +3453,69 @@ fn test_invalid_exit_or_return() {
     }
 }
 
+#[test]
+fn callx_unsupported_instruction_and_exceeded_max_instructions() {
```

Comment:
How is this different from the second part of `test_err_capped_before_exception()`?